nscplugin/src/main/scala/scala/scalanative/nscplugin/PrepNativeInterop.scala

```diff
@@ -252,6 +266,57 @@ abstract class PrepNativeInterop
     }
   }
 
+  /**
+   * Rewrite the rhs of `lazy val scalaProps` in trait `PropertiesTrait` to return a pre-populated
+   * `java.util.Properties` if the resource to read is `/library.properties`.
+   * @param original  The original `DefDef`
+   * @param freshName A function that generates a fresh name
+   * @return The new (typed) rhs of the given `DefDef`.
+   */
+  private def ShortCircuitLibraryProperties(
```

Comment:
Why is this method capitalized? Lets make it lower-case for consistency.

Reply:
To follow the same convention as `private def ScalaEnumValName` that is defined a few lines before. Should I still change it?

Reply:
Lets switch to lower-case for the start of the name of the all methods in there.